public/app/features/inspector/InspectDataTab.test.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { ComponentProps } from 'react';
+import { FieldType } from '@grafana/data';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { InspectDataTab } from './InspectDataTab';
+
+const createProps = (propsOverride?: Partial<ComponentProps<typeof InspectDataTab>>) => {
+  const defaultProps = {
+    isLoading: false,
+    options: {
+      withTransforms: false,
+      withFieldConfig: false,
+    },
+    data: [
+      {
+        name: 'First data frame',
+        fields: [
+          { name: 'time', type: FieldType.time, values: [100, 200, 300] },
+          { name: 'name', type: FieldType.string, values: ['uniqueA', 'b', 'c'] },
+          { name: 'value', type: FieldType.number, values: [1, 2, 3] },
+        ],
+        length: 3,
+      },
+      {
+        name: 'Second data frame',
+        fields: [
+          { name: 'time', type: FieldType.time, values: [400, 500, 600] },
+          { name: 'name', type: FieldType.string, values: ['d', 'e', 'g'] },
+          { name: 'value', type: FieldType.number, values: [4, 5, 6] },
+        ],
+        length: 3,
+      },
+    ],
+  };
+
+  return Object.assign(defaultProps, propsOverride) as ComponentProps<typeof InspectDataTab>;
+};
+
+describe('InspectDataTab', () => {
+  describe('when panel is not passed as prop (Explore)', () => {
+    it('should render InspectDataTab', () => {
+      render(<InspectDataTab {...createProps()} />);
+      expect(screen.getByLabelText(/Panel inspector Data content/i)).toBeInTheDocument();
+    });
+    it('should render Data Option row', () => {
+      render(<InspectDataTab {...createProps()} />);
+      expect(screen.getByText(/Data options/i)).toBeInTheDocument();
+    });
+    it('should show available options', () => {
+      render(<InspectDataTab {...createProps()} />);
+      const dataOptions = screen.getByText(/Data options/i);
+      fireEvent.click(dataOptions);
```

Comment:
If you want to simulate a user clicking on something you should use userEvent.click instead.
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-testing-libraryuser-event

Reply:
Thanks for sharing!